src/middlewared/middlewared/plugins/device_/device_info_linux.py

```diff
@@ -100,7 +100,8 @@ def retrieve_disks_data(self):
         disks_cp = subprocess.run(
             ['lsblk', '-OJdb'],
             stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE
+            stderr=subprocess.PIPE,
+            errors="ignore"
```

Comment:
```suggestion
            errors='ignore'
```

We use single quotes in this whole file